### ISSUE TITLE
Bug Octavia: Missing fields for load balancer's pool member: ProjectID, CreatedAt, UpdatedAt, OperatingStatus, Backup, MonitorAddress, MonitorPor

### ISSUE THREAD

**Issue by JackKuei** [gophercloud/gophercloud#1254]
When listing the pool member detail, some fields in member are missing when using octavia.
Possibly related to https://github.com/gophercloud/gophercloud/issues/1149

Error handling code was removed for brevity
```
	client, err := openstack.NewLoadBalancerV2(provider, gophercloud.EndpointOpts{Region: "dev"})
	allLBPages, err := loadbalancers.List(client, loadbalancers.ListOpts{}).AllPages()
	allLB, err := loadbalancers.ExtractLoadBalancers(allLBPages)
	allPoolPages, err := pools.List(client, pools.ListOpts{}).AllPages()
	allMembers, err := pools.ExtractPools(allPoolPages)
	for _, member := range allMembers {
		allPages, err := pools.ListMembers(client, member.ID, pools.ListMembersOpts{}).AllPages()
		allPoolMembers, err := pools.ExtractMembers(allPages)
		for _, member := range allPoolMembers {
                        // MISSING FIELDS FOR MEMBER
                        // MISSING member.OperatingStatus
                        // MISSING member.MonitorAddress
		}
	}
```

**Comment by JackKuei** [comment]
Possibly Invalid. Will Reopen if it is an actual issue.

**Comment by jtopjian** [comment]
Implemented in #1255
